src/ServiceControl.Audit/Infrastructure/AuditMetrics.cs

```diff
@@ -0,0 +1,10 @@
+namespace ServiceControl.Audit;
+
+using System.Diagnostics.Metrics;
+
+static class AuditMetrics
+{
+    public const string MeterName = "Particular.ServiceControl";
+    public static readonly Meter Meter = new(MeterName, "0.1.0");
+    public static readonly string Prefix = "particular.servicecontrol.audit";
+}
```

Comment:
I think this should better define its intend:

- No need for Audit in the name as the namespace already is clear
- Its for OpenTelemetry
- These are constants

```suggestion
static class TelemetryGlobals
{
    public const string Source = "Particular.ServiceControl";
    public const string Scope = "particular.servicecontrol.audit";
    public static readonly Meter Meter = new(MeterName, "0.1.0");
}
```